### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Review by bizley — APPROVED** [review]

File: framework/db/mssql/Schema.php [hunk 1]
```diff
@@ -565,7 +565,7 @@ protected function findForeignKeys($table)
         }
 
         // please refer to the following page for more details:
-        // http://msdn2.microsoft.com/en-us/library/aa175805(SQL.80).aspx
+        // https://msdn2.microsoft.com/en-us/library/aa175805(SQL.80).aspx
```
Comment (bizley) [inline framework/db/mssql/Schema.php:568]: This link doesn't work

File: framework/captcha/SpicyRice.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-* **License:** SIL Open Font License (OFL), version 1.1, [notes and FAQ](http://scripts.sil.org/OFL)
+* **License:** SIL Open Font License (OFL), version 1.1, [notes and FAQ](https://scripts.sil.org/OFL)
 
 ## Links
 
 * [Astigmatic](http://www.astigmatic.com/)
-* [Google WebFonts](http://www.google.com/webfonts/specimen/Spicy+Rice)
-* [fontsquirrel.com](http://www.fontsquirrel.com/fonts/spicy-rice)
-* [fontspace.com](http://www.fontspace.com/astigmatic-one-eye-typographic-institute/spicy-rice)
+* [Google WebFonts](https://www.google.com/webfonts/specimen/Spicy+Rice)
```
Comment (bizley) [inline framework/captcha/SpicyRice.md:9]: This link doesn't work

File: framework/caching/ZendDataCache.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
 /**
  * ZendDataCache provides Zend data caching in terms of an application component.
  *
- * To use this application component, the [Zend Data Cache PHP extension](http://www.zend.com/en/products/server/)
+ * To use this application component, the [Zend Data Cache PHP extension](https://www.zend.com/en/products/server/)
```
Comment (bizley) [inline framework/caching/ZendDataCache.php:13]: This link doesn't work

**Review by bizley — CHANGES_REQUESTED** [review]

File: framework/captcha/SpicyRice.md (hunk 2 above)
Comment (Arhell) [reply framework/captcha/SpicyRice.md]: update

File: framework/db/mssql/Schema.php (hunk 1 above)
Comment (Arhell) [reply framework/db/mssql/Schema.php]: remove htttps

File: framework/caching/ZendDataCache.php (hunk 3 above)
Comment (Arhell) [reply framework/caching/ZendDataCache.php]: update thanks

**Review by bizley — APPROVED** [review]

**Comment by bizley** [comment]
👍🏻

**Pull request merged**